assets/js/base/context/cart-checkout/cart/index.js

```diff
@@ -22,6 +23,7 @@ export const CartProvider = ( { children, redirectUrl } ) => {
 			<ShippingDataProvider>
 				<PaymentMethodDataProvider>
 					{ children }
+					<CartProcessor />
```

Comment:
Similar pattern to checkout in that it consumes contexts further up the tree. In this case it listens for shipping and billing address updates and dispatches updates to the API on change.